CONTRIBUTING.md

```diff
@@ -99,6 +99,59 @@ This also sets up gulp and Jekyll to watch for file changes to the `/docs`
 and `/src` directories and rebuilds the website accordingly. The command is
 aliased by `npm start`
 
+### Committing files when updating the `/src` directory
+
+When you run `npm start` to preview the website locally, many files are
```

Comment:
If possible (that is, if it makes sense in this context), I'd change this to active voice:

"When you run npm start to preview the website locally, you generate many files that are tracked by Git."